packages/css-framework/src/components/_phase-banner.scss

```diff
@@ -0,0 +1,25 @@
+.rn-phase-banner {
+  display: block;
+  background-color: color(primary, 100);
+  padding: spacing(2) spacing(6);
+  &--alpha {
```

Comment:
are these needed? There's `display: block;` already on the `.rn-phase-banner`

Reply:
Whoops. That was left over from earlier. That's what happens when you start something one day and finish it the other.